### PR TITLE
upgrade version

### PR DESCRIPTION
**Pull request by yaelballa**
@shlomisut

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -27,12 +27,11 @@ repositories {
...
 }
 
-
 dependencies {
 
-    ext.dropwizardVersion = '0.8.1'
+    ext.dropwizardVersion = '0.8.4'
     ext.springIoPlatformVersion = '1.1.2.RELEASE'
-    ext.jerseyVersion = '1.19'
+    ext.jerseyVersion = '2.22.1'
```
Comment (shlomisut) [inline build.gradle]: why do we need direct jersey here? isn't it part of DW-core anyhow ?

File: build.gradle [hunk 2]
```diff
@@ -132,9 +129,9 @@ uploadArchives {
             // when we're ready to publish to Maven Central, use this repository instead
             // (make sure a project has been created for us first!)
             // see: http://central.sonatype.org/pages/ossrh-guide.html#create-a-ticket-with-sonatype
-            repository(url: "https://oss.sonatype.org/service/local/staging/deploy/maven2/") {
-                authentication(userName: project.property("sonatypeUsername"), password: project.property("sonatypePassword"))
-            }
+            //repository(url: "https://oss.sonatype.org/service/local/staging/deploy/maven2/") {
```
Comment (shlomisut) [inline build.gradle]: see my comment on the previous PR

**Comment by yaelballa** [comment]
@shlomisut 
I uploaded manually to kenshoo artifactory

http://artifactory.kenshoo-lab.com/artifactory/webapp/browserepo.html?3&pathId=PyPI-local:io/github/fallwizard/Fallwizard/0.8.4.0/Fallwizard-0.8.4.0.jar

**Comment by shlomisut** [comment]
looks good, sqaush & I'll merge

**Comment by yaelballa** [comment]
done

**Pull request merged**